src/runrms/_forward_model.py

```diff
@@ -55,6 +56,17 @@ def validate_pre_experiment(self, fm_step_json: ForwardModelStepJSON) -> None:
         if not ok:
             raise ForwardModelStepValidationError(f"FMRMSConfig: {err}")
 
+        if "<RMS_PYTHONPATH>" in self.private_args:
+            ForwardModelStepWarning.warn(
+                "Please remove <RMS_PYTHONPATH> from your RMS step configuration. "
+                "It does nothing."
```

Comment:
Other messages that comes from Ert are more similar in style with this;

"Remove unused keyword <RMS_PYTHONPATH> from your RMS step configuration. It has no effect."

Provided as an alternative, also fine as is.